cli/internal/fs/turbo_json_test.go

```diff
@@ -164,6 +163,10 @@ func Test_ReadTurboConfig_EnvDeclarations(t *testing.T) {
 	assert.EqualValues(t, sortedArray(pipeline["task9"].EnvVarDependencies), sortedArray([]string{"A"}))
 	assert.EqualValues(t, sortedArray(pipeline["task10"].EnvVarDependencies), sortedArray([]string{"A"}))
 	assert.EqualValues(t, sortedArray(pipeline["task11"].EnvVarDependencies), sortedArray([]string{"A", "B"}))
+
+	// check global env vars also
+	assert.EqualValues(t, sortedArray([]string{"FOO", "BAR", "BAZ"}), sortedArray(turboJSON.EnvVarDependencies))
+	assert.EqualValues(t, sortedArray([]string{"somefile.txt"}), sortedArray(turboJSON.GlobalFileDependencies))
```

Comment:
There are probably more test cases I could add here (similar to above), but it would require more `testdata/` folders since there is only one globalEnv/globalDeps key. If there are additional values I can add without doing that, let me know, but will defer creating more testdata folders